### DISCUSSION TITLE
Where i can find Primereact core css docs?

### DISCUSSION THREAD

**Discussion by skj-skj** [primefaces/community#1165] [PrimeReact] [answered]
I went to https://primereact.org/ and saw a component is using p-float-label css class to implement floating label. I want to see what other classes are there to use, but was not able to find any docs or guide.

**Comment by melloware** [comment, marked as answer]
Most of the core styles like `p-float-label` are defined here: https://github.com/primefaces/primereact/blob/master/components/lib/componentbase/ComponentBase.js
